UI/OptionsWnd.cpp

```diff
@@ -692,6 +692,8 @@ void OptionsWnd::CompleteConstruction() {
     BoolOption(current_page, 0, "autosave.multiplayer",   UserString("OPTIONS_MULTIPLAYER"));
     IntOption(current_page,  0, "autosave.turns",         UserString("OPTIONS_AUTOSAVE_TURNS_BETWEEN"));
     IntOption(current_page,  0, "autosave.limit",         UserString("OPTIONS_AUTOSAVE_LIMIT"));
+    BoolOption(current_page, 0, "autosave.initial-turn",  UserString("OPTIONS_AUTOSAVE_INITIAL_TURN"));
+    BoolOption(current_page, 0, "autosave.last-turn",     UserString("OPTIONS_AUTOSAVE_LAST_TURN"));
```

Comment:
There is a distinction in the server side code, but the programmer can see that the save is generated in the human client FSM `WaitingForGameStart` when the first turn game data is extracted.

I don't think there is a distinction to a player.

I forgot to change the variable names in the non-UI code in `HumanClientApp`, so I have re-written the last commit.